src/components/Panel/__tests__/Panel.spec.js

```diff
@@ -3,136 +3,38 @@
  * @copyright IBM Security 2019
  */
 
-import { shallow, mount } from 'enzyme';
+import { mount } from 'enzyme';
 import React, { Fragment } from 'react';
 
-import { label } from '../_mocks_';
+import { label, onClick } from '../_mocks_';
 
-import {
-  Button,
-  Panel,
-  PanelContainer,
-  PanelContent,
-  PanelController,
-} from '../../..';
-
-import { namespace } from '../PanelContainer';
+import { Button, Panel, PanelContent } from '../../..';
 
 describe('Panel', () => {
-  let panel;
-  let panelActive;
-
-  const { fn } = jest;
-
-  const onClose = fn(() => {
-    if (panelActive) {
-      panelActive = false;
-    }
-  });
-
-  const onOpen = fn(() => {
-    if (!panelActive) {
-      panelActive = true;
-    }
-  });
-
-  const panelElement = (
-    <Panel
-      onClose={onClose}
-      onOpen={onOpen}
-      shouldPanelOpen={!panelActive}
-      render={({ active, toggleActive }) => (
-        <Fragment>
-          <Button onClick={toggleActive}>{label}</Button>
-          <PanelController active={active}>
-            <PanelContainer
-              title={label}
-              subtitle={label}
-              closeButton={{ label, onClick: toggleActive }}
+  it('renders', () => {
+    const wrapper = mount(
+      <Panel
+        title={label}
+        subtitle={label}
+        closeButton={{ onClick }}
```

Comment:
```suggestion
        closeButton={onClick}
```

Reply:
I don't even know how this happened? 🤣

Reply:
I think it was already in the `PanelV2` so the question should be -- how/why did I do this? 😆